src/components/auth/AuthGuard.tsx

```
@@ -1,20 +1,35 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { useAuth } from '@/hooks/use-auth';
 import { auth } from '@/lib/firebase';
 import { AuthScreen } from './AuthScreen';
+import { auth } from '@/lib/firebase';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'auth'. Consider to delete it or rename it.

'auth' is defined here:


(lint/suspicious/noRedeclare)